eng/scripts/Analyze-Code.ps1

```diff
@@ -34,7 +34,7 @@ if ($CheckWasm) {
   Invoke-LoggedCommand "cargo +$Toolchain clippy --target=wasm32-unknown-unknown --workspace --keep-going --no-deps"
 }
 
-Invoke-LoggedCommand "cargo +$Toolchain doc --workspace --no-deps"
```

Comment:
I'm referencing `into_xml_body` in a doc comment, which only exists in the `xml` feature. IMO, our docs should build with all reasonable features, and include "Requires `xyz` feature" notices as necessary (docs.rs can do this, and I've enabled it in `azure_data_cosmos`). So, I figured the best fix was to have our analyze pass run the docs build with `--all-features`. The alternative, which I'm open too if we want to avoid this change, is to remove the reference to `into_xml_body`.

Reply:
I want to enable `--all-features` anyway in every run, but the `e2e_test` and your `keyauth` feature prevent that. We should eliminate those or find another way.

Reply:
The key auth feature shouldn't be blocking that. It's fully additive and intended to be tested in unit tests.